sphinx/locale/__init__.py

```diff
@@ -229,11 +229,17 @@ def lazy_gettext(string):
 if PY3:
     def _(message):
         # type: (unicode) -> unicode
-        return translators['sphinx'].gettext(message)
+        try:
+            return translators['sphinx'].gettext(message)
+        except KeyError:
+            return message
```

Comment:
Why this change is needed?
Did the change of the initialization timing have any effect?